Add documentation for units (#8)
iterorganization/Fusion-Conventions: docs/index.md

### Units

The `units` attribute is required for all variables that represent dimensional

> quantities. The `units` attribute is permitted (with a value of `1`) but not

Reviewer: ```suggestion
For dimensionless quantities, the units attribute is optional. When specified, it may have a value of 1.
```

> quantities. The `units` attribute is permitted (with a value of `1`) but not
> required for dimensionless quantities.

Reviewer: ```suggestion
quantities. For dimensionless quantities, the `units` attribute is optional. When specified, it may have a value of `1`.
```
